x-pack/plugins/monitoring/public/components/tooltip/index.js

```diff
@@ -5,26 +5,20 @@
  */
 
 import React from 'react';
-import { Tooltip as PuiTooltip } from 'pivotal-ui/react/tooltip';
-import { OverlayTrigger as PuiOverlayTrigger } from 'pivotal-ui/react/overlay-trigger';
+import { EuiToolTip } from '@elastic/eui';
 
 export const Tooltip = props => {
-  const tooltip = (
-    <PuiTooltip>{ props.text }</PuiTooltip>
-  );
-
   return (
-    <PuiOverlayTrigger
-      placement={props.placement}
-      trigger={props.trigger}
-      overlay={tooltip}
+    <EuiToolTip
+      content={props.text}
+      position={props.placement}
     >
       <span className="overlay-trigger">
```

Comment:
I wonder if we can throwaway this CSS too and just put in `{ props.children }` here directly?

Reply:
Yes both of these make sense! I think we can just remove it